checkpoint/cp-resource-management/templates/addons/flow-logs/daemonset.yaml

```diff
@@ -0,0 +1,100 @@
+  template:
+    metadata:
+      annotations:
+        agentVersion: {{ .Values.addons.flowLogs.daemonset.image.tag | quote }}
+        seccomp.security.alpha.kubernetes.io/pod: {{ .Values.podAnnotations.seccomp }}
+        {{- if .Values.podAnnotations.apparmor }}
+        container.apparmor.security.beta.kubernetes.io/{{ .Release.Name }}-flow-logs: 
+{{ toYaml .Values.podAnnotations.apparmor | indent 10 }}
+        {{- end }}
+      labels:
+        app.kubernetes.io/name: {{ .Release.Name }}-flow-logs
+        app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    spec:
+      {{- if .Values.rbac.pspEnable }}
+      serviceAccountName: {{ .Release.Name }}-flow-logs
+      {{- end }}  
+      {{- if .Values.addons.flowLogs.daemonset.nodeSelector }}
+      nodeSelector:
+{{ toYaml .Values.addons.flowLogs.daemonset.nodeSelector | indent 8 }}
+      {{- end }}
+      {{- if .Values.addons.flowLogs.daemonset.affinity }}
+      affinity:
+{{ toYaml .Values.addons.flowLogs.daemonset.affinity | indent 8 }}
+      {{- end }}
+      {{- if .Values.addons.flowLogs.daemonset.tolerations }}
+      tolerations:
+{{ toYaml .Values.addons.flowLogs.daemonset.tolerations | indent 8 }}
+      {{- end }}
+      {{- if .Values.addons.flowLogs.daemonset.resources }}        
+        resources:          
+{{- toYaml .Values.addons.flowLogs.daemonset.resources | indent 8 }}
+      {{- end }}        
+      containers:
+        - name: {{ .Release.Name }}-flow-logs
+          image: "{{ .Values.addons.flowLogs.daemonset.image.repository }}:{{ .Values.addons.flowLogs.daemonset.image.tag }}"
+          imagePullPolicy: {{ .Values.addons.flowLogs.daemonset.image.pullPolicy }}
+          securityContext:
+            privileged: true
+          volumeMounts:
+            - mountPath: /var/run/netns
+              name: netns-volume
+              mountPropagation: HostToContainer
+            - mountPath: /var/run/docker.sock
+              name: docker-sock-volume
+            - mountPath: /etc/secret-volume
+              name: secret-volume
+          command: ["/bin/sh"]
+          args: ["-c", "/flow_logs_user"]
+          env:
+            - name: NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+            - name: DOME9_URL
+              value: "https://api-cpx.dome9.com"
+            - name: CP_KUBERNETES_CLUSTER_ID
+              valueFrom:
+                configMapKeyRef:
+                  name: {{ .Release.Name }}-cp-cloudguard-configmap
+                  key: clusterID
+            - name: CP_FLOW_LOGS_REPORT_PERIOD
+              value: "30"
+            - name: CP_FLOW_LOGS_DISCOVERY_PERIOD
+              value: "60"
+            - name: LOG_LEVEL
+              value: "debug"
+            - name: LOCAL_DEBUG
+              value: "1"   
+            - name: COMPRESS
+              value: "false"
+      terminationGracePeriodSeconds: 30      
```

Comment:
delete. It was left since Vladimir yaml, I see 30 seconds is the [default](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#pod-v1-core) (search for terminationGracePeriodSeconds)